Swift_Basics.playground/Contents.swift

```diff
@@ -0,0 +1,587 @@
+    chars.append(firstChar)
+    chars.append(secondChar)
+    return String(chars)
+}
+print(moveCharsToEnd("Swift"))
+
+//Write a Swift program to test if a given string starts with "Sw"
+func startsWith(_ input: String) -> Bool {
+    if input.hasPrefix("Sw") {
+        return true
+    } else {
+        return false
+    }
+}
+print(startsWith("Swift"))
+
+//Write a Swift program to create a new string made of the first and last n chars from a given string. The string length will be at least n.
+func makeStrOfNChars(_ input: String, _ lastNchars: Int) -> String {
+    let lastNcharsOfFirstpart = String([Character](input).prefix(lastNchars))
+    let lastNcharsOfSecondpart = String([Character](input).suffix(lastNchars))
+    return lastNcharsOfFirstpart + lastNcharsOfSecondpart
+}
+print(makeStrOfNChars("Simform Solutions", 3))
+
+//Write a Swift program to check whether the first element and the last element of a given array of integers are equal. The array length must be 1 or more
+func checkEqualElements(_ inputArray: [Int]) -> Bool {
+    guard inputArray.count > 0 else {
+        return false
+    }
+    if inputArray.first == inputArray.last {
+        return true
+    } else {
+        return false
+    }
+}
+print(checkEqualElements([1, 2, 3, 1]))
+
+//Write a Swift program to create a new array, taking first two elements from a given array of integers. If the length of the given array is less than 2 use the single element of the given array
+func firstTwoElements(_ inputArray: [Int]) -> [Int] {
+    guard inputArray.count > 1 else {
+        return inputArray
+    }
+    return Array(inputArray.prefix(2))
+}
+print(firstTwoElements([0, 1, 2, 3]))
+
+//Write a Swift program to take one array and find out maximum number from that array
+var elements = [0, 50, 100, 20, 80, 150]
+var largestNum = elements.max()
+
+//Write a Swift program to take one array and sort it in descending order.
+let inputArray = [0, 5, 6, 2, 10]
+let reversedArray = Array(inputArray.sorted().reversed())
+print("Descending order: \(reversedArray)")
+
+//Print all the numbers from listOfNumbers that are divisible by at least one number from divisors
+var listOfNumbers = [1, 2, 3, 10, 100]
+var divisorArray = [2, 5]
+var output: String = ""
+for value in listOfNumbers {
+    for divisor in divisorArray {
+        if value % divisor == 0 {
+            output += "\(value),"
+            break
+        }
+    }
+}
+print(output.dropLast(1))
+//Print the powers of 2 that are less than or equal to N
+var maxNum = 100
+var inputNum = 1
+while inputNum < maxNum {
+    inputNum *= 2
+    if (inputNum < maxNum) {
+        print(inputNum)
+    }
+}
+
+let givenNum: Int = 3
+for _ in 1...givenNum {
+    for _ in 1...givenNum {
+        print("*", terminator: "")
+    }
+    print("\n")
+}
+
+for ipNum in 1..<5 {
+    for _ in 1...ipNum {
+        print("*", terminator: "")
+    }
+    print("")
+}
+
+let thirdInput: Int = 5
+for row in 1..<thirdInput {
```

Comment:
just change spacing in every for loop in whole file from for row in 1..<thirdInput to for row in 1 ..< thirdInput